2.evaluate_data/evaluate_data.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
 # initialize the correct shell for your machine to allow conda to work (see README for note on shell names)
-conda init bash
+eval "$(conda shell.bash hook)"
```

Comment:
I didn't find this in the README - am I missing something?

Also from a security standpoint it is generally best practice to avoid the use of `eval` in scripts

Reply:
Good to know, for some reason conda was yelling when I used `conda init bash`. Maybe there is a better way?